package-lock.json

```diff
@@ -22,6 +22,7 @@
         "@trpc/next": "10.37.1",
         "@trpc/react-query": "10.37.1",
         "@trpc/server": "10.37.1",
+        "@uidotdev/usehooks": "^2.4.1",
```

Comment:
do not install with carets (meaning version X and minor updates) -> always pin a specific version. when installing with npm, use `npm i -E` ("exact") instead of `npm i`.

Reply:
change to be done: remove ^ in package.json and `npm i` again & commit changes

Reply:
Done